usecases/demonstrator/ExperimentData/Examples/extractor.py

```diff
@@ -0,0 +1,48 @@
+import json as js
+import os
+import numpy as np
+with open("calo_example.csv","r") as f:
```

Comment:
As mentioned above, this script is a hard coded example for a single file.
This does not seem practical.
Rewrite it as a funciton that take the flie as input, might be already sufficient.